modules/oe_content_publication/config/post_updates/00003_create_fields/field.field.node.oe_publication.body.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.body
+    - node.type.oe_publication
+  module:
+    - text
+id: node.oe_publication.body
+field_name: body
+entity_type: node
+bundle: oe_publication
+label: Body
+description: ''
+required: false
+translatable: false
```

Comment:
According to the description, it should be translatable. Did I miss something?

Reply:
I set it to translatable.

Reply:
@georgebaev please revert back. @sergepavle we agreed to set `translatable: true` in the field.storage.* configurations only. field.field.* always has `translatable: false`.

Reply:
I reverted the setting.